diff --git a/bin/dune b/bin/dune
index ec19a0c..69ca5c3 100644
--- a/bin/dune
+++ b/bin/dune
@@ -3,6 +3,7 @@
  (modules Main)
  (public_name osnap)
  (flags
+  (-ccopt -static)
   (:standard -w +A-42-44-48-66-40 -warn-error +A-3-44-66-40))
  (ocamlopt_flags -O3)
  (libraries OSnap console.lib pastel.lib cmdliner)
@@ -14,6 +15,7 @@
  (modules DownloadChromium)
  (public_name downloadChromium)
  (flags
+  (-ccopt -static)
   (:standard -w +A-42-48-66-40 -warn-error +A-3-44-66-40))
  (libraries OSnap camlzip httpaf httpaf-lwt-unix lwt)
  (preprocess
